stubs/requests/requests/compat.pyi

```diff
@@ -16,3 +17,10 @@ from urllib.request import getproxies as getproxies, parse_http_list as parse_ht
 is_py2: bool
 is_py3: bool
 has_simplejson: bool
+
+builtin_str: TypeAlias = str  # noqa: Y042
+# str = str omitted
+# bytes = bytes omitted
+basestring: tuple[type, type]
+numeric_types: tuple[type, type]
+integer_types: tuple[type]
```

Comment:
I feel like the exact lengths of these is an implementation detail that might change in the future

```suggestion
basestring: tuple[type, ...]
numeric_types: tuple[type, ...]
integer_types: tuple[type, ...]
```